cfpb/design-manual
# Update uglify to use harmony branch

`grunt-contrib-uglify` does not support ES6 minification because the underlying UglifyJS dependency doesn't. The recommended workaround is to [use the `harmony` branch of `grunt-contrib-uglify`](https://github.com/mishoo/UglifyJS2/issues/448#issuecomment-312352150).

## Changes

- Changes `grunt-contrib-uglify` to use the `harmony` branch instead of the release.

## Testing

- Run `npm install`.
- Run `grunt uglify` and it should not choke.

## Todos

- The project should really be updated to use babel/webpack and uglify through that pipeline.
